pkgs/top-level/ocaml-packages.nix

```diff
@@ -297,10 +299,14 @@ let
 
     ocamlfuse = callPackage ../development/ocaml-modules/ocamlfuse { };
 
+    ocaml_gettext = callPackage ../development/ocaml-modules/ocaml-gettext { };
```

Comment:
You can use `pkgs.gettext` or `inherit (pkgs) gettext` in the `callPackage` argument to overcome this issue.